pkg/config/config.go

```diff
@@ -32,6 +39,19 @@ func (s *Serialized) ToController() (*Controller, error) {
 	if cfg.Interval <= 0 {
 		return nil, fmt.Errorf("interval must be a non-negative duration")
 	}
+
+	if len(s.SmartProxyConfig.PollTime) > 0 {
```

Comment:
or `s.SmartProxyConfig.PollTime != ""` if I understand correctly

Reply:
Yes, you are right, done